docs/book/src/topics/capibmadm/powervs/port.md

```diff
@@ -41,4 +42,29 @@ IBMCLOUD_API_KEY: IBM Cloud API key.
 ```shell
 export IBMCLOUD_API_KEY=<api-key>
 capibmadm powervs port delete --port-id <port-id> --network <network-name/network-id> --service-instance-id <service-instance-id> --zone <zone>
+```
+
+### 3. capibmadm powervs port create
+
+#### Usage:
+Create PowerVS network port.
+
+#### Environmental Variable:
+IBMCLOUD_API_KEY: IBM Cloud API key.
+
+#### Arguments:
+--service-instance-id: PowerVS service instance id.
+
+--zone: PowerVS service instance zone.
+
+--network: Network ID/ Network Name.
+
+--description: Description of the port.
+
+--ip-address: The requested IP address of this port
+
+#### Example:
+```shell
+export IBMCLOUD_API_KEY=<api-key>
+capibmadm powervs port create --network <netword-id/network-name> --description <description> --service-instance-id <service-instance-id> --zone <zone>
```

Comment:
Will you be able to rearrange the contents in this file to have 
1. Create
2. Delete
3. List  
commands?